net-mgmt/pfSense-pkg-softflowd/files/usr/local/pkg/softflowd.inc

```diff
@@ -46,8 +47,9 @@ function sync_package_softflowd() {
 			if (empty($interface)) {
 				continue;
 			}
+			
 			$start .= "\n\t/usr/local/sbin/softflowd ";
-			$start .= " -i " . escapeshellarg($interface);
+			$start .= " -i " . $if_index++ . ":" . escapeshellarg($interface);
```

Comment:
Does that actually work properly? It will construct the string as, for example with igb1: `-i 1:'igb1'`. It looks like it might be better as: 
`$start .= " -i " . escapeshellarg($if_index++ . ":" . $interface);`

Which would result in `-i '1:igb1'

Reply:
Good catch; I clearly hacked that line together too quickly. It did however work with just the interface between the quotes. The "getopt" call in softflowd actually strips out the quotes (even the one following the colon right before the interface name); that really surprised me! Nevertheless, it was definitely an error on my part, looks better now.